.drone-v1.yml

```diff
@@ -0,0 +1,243 @@
+---
+kind: pipeline
+type: kubernetes
+name: build
+
+steps:
+  - name: build project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    commands:
+      - SPRING_PROFILES_ACTIVE='development, local'
```

Comment:
Can we move all of these into the `environment:` block?

Reply:
Done 👍🏼